pkg/client/push_manifest.go

```diff
@@ -0,0 +1,58 @@
+package client
+
+import (
+	"github.com/pkg/errors"
+)
+
+type PushManifestOptions struct {
+	// Image index we want to update
+	IndexRepoName string
+
+	// Index media-type
+	Format string
+
+	// true if we want to publish to an insecure registry
+	Insecure bool
+
+	// true if we want the index to be deleted from local storage after pushing it
+	Purge bool
+}
+
+// PushManifest implements commands.PackClient.
+func (c *Client) PushManifest(opts PushManifestOptions) (err error) {
+	idx, err := c.indexFactory.LoadIndex(opts.IndexRepoName)
+	if err != nil {
+		return
+	}
+
+	// TODO pass through the options
```

Comment:
This is important, I think after our imgutil refactoring, we didn't accept any option for pushing. That's why I commented all the code

Reply:
Done